src/GenericFactory.sol

```diff
@@ -53,7 +53,12 @@ contract GenericFactory is IGenericFactory, Owned {
 
         bytes memory lInitCode;
         uint256 lFreeMem;
-        assembly {
+
+        // SAFETY:
+        // This assembly block is memory safe because it simply loads the next free memory slot,
+        // gets the next free memory slot by adding the correct number of bytes (0x20),
+        // and does not overwrite previous memory
```

Comment:
"This assembly block is memory safe because" is implied and does not need to be re-written every time. I propose:

```solidity
// SAFETY:
Writes 1 word of new memory, updates the pointer before returning
```

Reply:
okay fixed